build/index.html

```diff
@@ -8,6 +8,8 @@
     <link rel="shortcut icon" id="favicon" type="image/ico" href="images/favicon-wim.png" />
 
     <script src="scripts/vendor.js"></script>
+    <script src="scripts/json2csv.js"></script>
```

Comment:
Was this added on purpose? The .js files in the 'scripts' folder get added together in the 'combined.js' file in the build so this shouldn't be necessary

Reply:
ah! No. I did add it there and thought I deleted it... looks like I didn't. One sec